### PR TITLE
Add professional services page

### LINKED ISSUE #1148: Professional services website page
### Description

The Sales and Customer Success teams are often asked whether FlowFuse provides Professional Services or flow engineering. It may be helpful to have a page on the website that mentions this capability and that it is discretionary, based on significant commercial engagement, a strategic project, or new vertical discovery, etc.

### Epic/Story

_No response_

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Created the /professional-services page

## Related Issue(s)

closes #1148

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
Hi @iskerrett I started working on the page. I still need to:
- Add it to the main nav, under the company menu.
- Update the form (let me know when you have it and all add it).

Add this: `/professional-services/` at the end of the preview URL to see it 😉

**Comment by iskerrett** [comment]
Here is the embeded javascript for the form.  

<script charset="utf-8" type="text/javascript" src="//js-eu1.hsforms.net/forms/embed/v2.js"></script>
<script>
  hbspt.forms.create({
    region: "eu1",
    portalId: "26586079",
    formId: "b8ae3144-7eb2-4f5a-a9f8-f99e5c7bb07f"
  });
</script>

File: src/professional-services.njk [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+align: center
+layout: layouts/page.njk
+title: Professional Services
+description: <p>FlowFuse partner with our customers to ensure their successful deployment of FlowFuse and Node-RED. Discover our professional service offerings.</p>
```
Comment (iskerrett) [inline src/professional-services.njk:5]:
```suggestion
description: <p>FlowFuse partners with our customers to ensure their successful deployment of FlowFuse and Node-RED. Discover our professional service offerings.</p>
```

File: src/professional-services.njk [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            {% image "./images/services/engineering-and-architecture-services.png", "Image of hands typing on laptop working on Node-RED flows", [357] %}
+          </div>
+          <div class="prose">
+            <h3 class="flex items-center">
+              <div class="flex flex-col leading-8 mt-3 text-center md:text-left w-full">
+                Engineering and Architecture Services
+              </div>
+            </h3>
+            <p class="mb-0">
+              FlowFuse are experts in Node-RED development. We can help your team create Node-RED flows that meet your requirements or we can conduct an architecture review of existing flows to align with Node-RED best practices. Flow engineering is available to Enterprise tier customers of FlowFuse.
```
Comment (iskerrett) [inline src/professional-services.njk:22]:
```suggestion
              FlowFuse employees are experts in Node-RED development. We can help your team create Node-RED flows that meet your requirements or we can conduct an architecture review of existing flows to align with Node-RED best practices. Flow engineering is available to Enterprise tier customers of FlowFuse.
```

**Review by iskerrett — APPROVED** [review]

File: src/professional-services.njk [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    algoliasearchNetlify({
+        appId: 'KWE0727TZS',
+        apiKey: '595ee07f069991380e9c3036cfad8e5e',
+        siteId: '00f8cf60-997f-4c4d-9427-a97924358648',
+        branch: 'live',
+        selector: 'div#algolia-search',
+        detached: false
+    });
+    }
+</script>
```
Comment (ZJvandeWeg) [inline src/professional-services.njk:85]: Can we create `algolia.njk` with this contents, and include it everywhere where it's required to keep our code DRY?

Comment (Yndira-FlowForge) [reply src/professional-services.njk]: Hi @ZJvandeWeg this wasn't necessary for this page, I just deleted it

**Pull request merged**